fdbserver/CoroFlow.actor.cpp

```diff
@@ -279,8 +278,6 @@ ACTOR void coroSwitcher( Future<Void> what, int taskID, Coro* coro ) {
 void CoroThreadPool::waitFor( Future<Void> what ) {
 	ASSERT (current_coro != main_coro);
 	if (what.isReady()) return;
-	//double t = now();
-	coroSwitcher( what, g_network->getCurrentTask(), current_coro );
```

Comment:
Should keep this call?

Reply:
Yes, that was a botched merge conflict resolution. Thanks for catching that.